server/package-lock.json

```diff
@@ -1,11448 +1,8 @@
 {
   "name": "health-career-access-program-server",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```

Comment:
why are we downgrading from lockfile version 2 to 1. I think your npm needs to be updated

Reply:
I did it in my local machine with node 14.17.0, nvm 6.14.13. Node 14-alpine is set as image.